README.md

```diff
@@ -20,7 +20,7 @@ Install the server node.js dependencies:
 Install the Ext JS framework for the application:
 
     $ cd client
-    $ sencha app install ~/sencha-sdks
+    $ sencha app upgrade ~/sencha-sdks/ext-<version of the sdk>
```

Comment:
Might be worth a word or two above as to why we use `upgrade` instead of `install`

Reply:
Well, install will always look for the version specified in app.json, while upgrade will use your version. Our current app.json expects 6.5.0, so if you place 6.5.1 or 6.5.2 there, it will show an error saying 6.5.0 is not available.